tap_github/user_streams.py

```diff
@@ -158,3 +158,51 @@ def post_process(self, row: dict, context: Optional[dict] = None) -> dict:
             ),
         ),
     ).to_dict()
+
+
+class UserContributedToStream(GitHubGraphqlStream):
+    """Defines 'UserContributedToStream' stream. Warning: this stream 'only' gets the first 100 projects (by stars)."""
+
+    name = "user_contributed_to"
+    query_path = "user.repositoriesContributedTo.nodes"
+    primary_keys = ["username"]
+    replication_key = None
+    parent_stream_type = UserStream
+    state_partitioning_keys = ["username"]
+    ignore_parent_replication_key = True
+
+    @property
+    def query(self) -> str:
+        """Return dynamic GraphQL query."""
+        return """
+          query userContributedTo($username: String!) {
+            user (login: $username) {
+              repositoriesContributedTo (first: 100 includeUserRepositories: true orderBy: {field: STARGAZERS, direction: DESC}) {
+                nodes {
+                  id
+                  nameWithOwner
+                  openGraphImageUrl
+                  stargazerCount
+                  owner {
+                    id
+                    login
+                  }
+                }
+              } 
+            }    
+          }
+        """
+
+    schema = th.PropertiesList(
+        th.Property("id", th.StringType),
```

Comment:
From what I [understand](https://docs.github.com/en/graphql/guides/using-global-node-ids), `id` in github's graphql context is the same as `nodeId` in REST. Would it be worth renaming the properties here to give a consistent naming in the output of the tap? REST has both `id` (an integer) and `node_id` (string).